compiler/src/dmd/func.d

```diff
@@ -322,6 +322,12 @@ extern (C++) class FuncDeclaration : Declaration
 
     GotoStatements* gotos;              /// Gotos with forward references
 
+    VarDeclarations* alignSectionVars;  /// local variables with alignment needs larger than stackAlign
+    version (MARS)
+    {
+        Symbol* salignSection;              /// pointer to aligned section, if any
+    }
```

Comment:
```suggestion
    version (MARS)
    {
        VarDeclarations* alignSectionVars;  /// local variables with alignment needs larger than stackAlign
        Symbol* salignSection;              /// pointer to aligned section, if any
    }
```
Otherwise bad things will happen in C++ compilers trying to access `fd->builtin`, and others.

Reply:
incorporated